scripts/templates/local-dev.yml.template

```diff
@@ -0,0 +1,235 @@
+      IRODS_PORT: '1247'
+      IRODS_DEFAULT_RESOURCE: 'hydroshareReplResc'
+      IRODS_HOME_COLLECTION: '/hydroshareZone/home/wwwHydroProxy'
+      IRODS_ZONE: 'hydroshareZone'
+      IRODS_USERNAME: 'wwwHydroProxy'
+      IRODS_AUTH: 'wwwHydroProxy'
+      IRODS_GLOBAL_SESSION: 'True'
+      REMOTE_USE_IRODS: 'True'
+      HS_USER_ZONE_HOST: 'users.local.org'
+      LINUX_ADMIN_USER_FOR_HS_USER_ZONE: 'hsuserproxy'
+      LINUX_ADMIN_USER_PWD_FOR_HS_USER_ZONE: 'hsuserproxy'
+      LINUX_ADMIN_USER_CREATE_USER_IN_USER_ZONE_CMD: '/home/hsuserproxy/create_user.sh'
+      LINUX_ADMIN_USER_DELETE_USER_IN_USER_ZONE_CMD: '/home/hsuserproxy/delete_user.sh'
+      HS_IRODS_PROXY_USER_IN_USER_ZONE: 'localHydroProxy'
+      HS_IRODS_USER_ZONE_DEF_RES: 'hydroshareLocalResc'
+      IRODS_ZONE: 'hydroshareZone'
+      HS_USER_IRODS_ZONE: 'hydroshareuserZone'
+      IRODS_ZONE_NAME: 'hydroshareZone'
+      IRODS_SERVER_ZONE_KEY: 'hydroshareZone_KEY'
+      IRODS_DATABASE_SERVER_HOSTNAME: 'data.local.org'
+      IRODS_USER_NAME: 'rods'
+      IRODS_PASSWORD: 'rods'
+    stdin_open: true
+    tty: true
+  users.local.org:
+    image: hydroshare/hs-irods:4.2.6
+    container_name: users.local.org
+    hostname: users.local.org
+    volumes:
+      - "iuser_root_vol:/root"
+      - "iuser_hspx_vol:/home/hsuserproxy"
+      - "iuser_iconf_vol:/etc/irods"
+      - "iuser_vault_vol:/var/lib/irods"
+      - "iuser_pgres_vol:/var/lib/postgresql/data"
+    environment:
+      SHARED_NEG_KEY: 'hydroshareZonehydroshareuserZone'
+      USE_IRODS: 'True'
+      IRODS_ROOT: '/tmp'
+      IRODS_ICOMMANDS_PATH: '/usr/bin'
+      IRODS_HOST: 'users.local.org'
+      IRODS_PORT: '1247'
+      IRODS_DEFAULT_RESOURCE: 'hydroshareReplResc'
+      IRODS_HOME_COLLECTION: '/hydroshareZone/home/wwwHydroProxy'
+      IRODS_ZONE: 'hydroshareZone'
+      IRODS_USERNAME: 'wwwHydroProxy'
+      IRODS_AUTH: 'wwwHydroProxy'
+      IRODS_GLOBAL_SESSION: 'True'
+      REMOTE_USE_IRODS: 'True'
+      HS_USER_ZONE_HOST: 'users.local.org'
+      LINUX_ADMIN_USER_FOR_HS_USER_ZONE: 'hsuserproxy'
+      LINUX_ADMIN_USER_PWD_FOR_HS_USER_ZONE: 'hsuserproxy'
+      LINUX_ADMIN_USER_CREATE_USER_IN_USER_ZONE_CMD: '/home/hsuserproxy/create_user.sh'
+      LINUX_ADMIN_USER_DELETE_USER_IN_USER_ZONE_CMD: '/home/hsuserproxy/delete_user.sh'
+      HS_IRODS_PROXY_USER_IN_USER_ZONE: 'localHydroProxy'
+      HS_IRODS_USER_ZONE_DEF_RES: 'hydroshareLocalResc'
+      HS_USER_IRODS_ZONE: 'hydroshareuserZone'
+      IRODS_ZONE_NAME: 'hydroshareuserZone'
+      IRODS_SERVER_ZONE_KEY: 'hydroshareuserZone_KEY'
+      IRODS_DATABASE_SERVER_HOSTNAME: 'users.local.org'
+      IRODS_USER_NAME: 'rods'
+      IRODS_PASSWORD: 'rods'
+    stdin_open: true
+    tty: true
+  postgis:
+    image: hydroshare/hs_postgres:9.5.0
+    container_name: postgis
+    hostname: postgis
+    volumes:
+      - "postgis_data_vol:/var/lib/postgresql/data"
+    ports:
+      - "5432:5432"
+    stdin_open: true
```

Comment:
see previous comment on 54322 an 5432 and local developer laptops. Match setting as needed, based on outcome of discussion